src/main/java/net/dv8tion/jda/core/entities/impl/WebhookImpl.java

```diff
@@ -155,6 +155,9 @@ public WebhookManager getManager()
     @Override
     public WebhookClientBuilder newClient()
     {
+        if (token == null)
+            throw new IllegalStateException("Webhooks without known tokens (such as those retrieved from Audit Logs) "
```

Comment:
add this to the documentation

Reply:
`getManager()` should fail in the same way.

Reply:
I think we should always throw an exception for a fake webhook for this method. I would suggest a `WebhookClientBuilder#newClient(String token)` but that would be confusing for non-fake webhooks.